device/dhcp/common/dhcpv6options.yaml

```diff
@@ -0,0 +1,387 @@
+        link_layer_address:
+          description: |-
+            The link-layer address is stored in canonical form, as described in RFC 2464.           
+          $ref: '#/components/schemas/Dhcpv6Options.LinkLayerAddress'
+          x-field-uid: 2
+
+    Dhcpv6Options.DuidEn:
+      description: >-
+        DUID assigned by vendor based on enterprise number.
+      type: object
+      properties:
+        enterprise_number:
+          description: |-
+            Vendor's registered private enterprise number as maintained by IANA.            
+          type: integer
+          format: uint32
+          x-field-uid: 1
+        identifier:
+          description: |-
+            The unique identifier assigned by the vendor..           
+          type: string
+          x-field-uid: 2
+
+    Dhcpv6Options.DuidLl:
+      description: >-
+        DUID based on Link Layer address. Hardware Type will be auto assigned to ethernet type.
+      type: object
+      properties:
+       link_layer_address:
+          description: |-
+            The link-layer address is stored in canonical form, as described in RFC 2464.           
+          $ref: '#/components/schemas/Dhcpv6Options.LinkLayerAddress'
+          x-field-uid: 1   
+    
+    Dhcpv6Options.DuidUuid:
+      description: >-
+        DUID embedded a Universally Unique IDentifier (UUID).
+      type: object
+      properties:
+       uuid:
+          description: |-
+            The Universally Unique IDentifier embedded in DUID.           
+          type: string
+          x-field-uid: 1
+
+    Dhcpv6Options.LinkLayerAddress:
+      description: >-
+        The link-layer address configured in DUID llt of DUID ll. If configured on client for server identifier then user must
+        specify the choice as value. 
+      type: object
+      properties:
+        choice:
+          description: >-
+            In auto mode the system generated value is set for this property, while if the choice is selected as value, 
+            a user configured value will be used for this property.
+          type: string
+          default: auto
+          x-field-uid: 1
+          x-enum:
+            auto:
+              x-field-uid: 1
+            value:
+              x-field-uid: 2
+        auto:
+          description: >-
+            The OTG implementation must provide a system generated value for this property.
+          type: string
+          format: mac
+          x-field-uid: 2
+        value:
+          description: >-
+            User must specify a value if mode is not auto.
+          type: string
+          format: mac
+          x-field-uid: 3
+
+    Dhcpv6Options.Iana:
+      description: >-
+        The Identity Association for non temporary Addresses option (IA_NA option) is used to carry an IA_NA, the 
+        parameters associated with the IA_NA, and the non-temporary addresses associated with the IA_NA.
+      type: object
+      properties:
+        iaid:
```

Comment:
What if the user specifies all or none of the fields. Is is okay that the other values become language defaults i.e. 0 ?

Reply:
iaid is made mandatory, t1 and t2 values can be 0 , added as default